pyproject.toml

```diff
@@ -5,7 +5,7 @@ line_length = 99
 skip_gitignore = true
 
 [tool.codespell]
-skip = './src/control/vendor,./src/control/go.sum,./.git,./src/rdb/raft,./build,./install,./venv,./src/control/vendor/'
+skip = './src/control/vendor,./src/control/go.sum,./.git,./src/rdb/raft,./build,./install,./venv,./src/control/vendor/,./src/control/security/testdata/certs/source.txt'
```

Comment:
This is really a workaround since I'm not sure how to regenerate the `.golden` file. I'll try to follow up with a proper fix but this gets the check passing in master again